verificarlo/verificarlo
# Fix vector instrumentation + minor fixes:

- Vector instrumentation was invalid for unsupported vector:
      * New implementation used directly the signatures of vfcwrapper
      	in the instrumentation pass. To do so, we merge source and
	vfcwrapper IR files befor applying the instrumentation pass.
      * Add test for checking vectorization instrumentation in
      	tests/test_vectorization
    - Fix compilation of libVFCFuncInstrument.cpp for llvm-4.0
    - Update tests:
      * Remove test.sh.in generations for test_bitmask_backend,
      	test_fortran and test_fortran_NAS} and used paths.sh instead
      * Update clean.sh for some tests
      * Update test_branch_instrumentation test to check if scalar and
      	vector instrumentation give the same result
      * Fix test_cpp when using llvm-4
